### DISCUSSION TITLE
Suggestion: I2C master for general gamepad/input data

### DISCUSSION THREAD

**Discussion by wiredopposite** [darthcloud/BlueRetro#900] [Ideas]
It might be beyond the scope of this project, but adding this would allow for much easier use on consoles with USB. By using an MCU like a Pico or AVR (or even 4 of them) as I2C slave, with its own mappings and HID info, using BlueRetro for PS3/4, Xbox OG/360/One, Switch, or even PC, wouldn't be very complicated.

**Comment by darthcloud** [comment]
It's in the plan, likely this will happen this year.
